addon/modifiers/did-intersect.js

```diff
@@ -8,6 +8,11 @@ export const DEFAULT_OBSERVER_OPTIONS = {};
 
 function cleanup(instance) {
   instance.unobserve.call(instance);
+  instance.observerManager.removeElement(instance.element);
+  instance.observerManager.removeElement(window);
```

Comment:
Why `window` cleanup here? This modifier doesn't explicitly subscribe to `window` at all.

Reply:
Yes, I know it's weird. I'll be taking this to the `interesction-observer-admin` next.
I'd hope that this addon won't need these changes actually, since the issue lies on the dependency it uses.

But since I've had this experimental patch already, I decided I'd open a PR to showcase the issue. And frankly, I didn't think I'd get your attention so fast :sweat_smile:

Reply:
Heh! Well, I may not have had a lot of time for active development, but I very much appreciate contributions and will do my best to be responsive (though I'm on vacation for a week starting Saturday, FYI). :)

I definitely think some fixes to the underlying dependency make sense in the context of this issue. If this is a workaround until you can get those fixes in, can you add a brief comment explaining why the workaround is needed, and then we can remove it once you land fixes upstream?